app/views/hyrax/my/_work_action_menu.html.erb

```diff
@@ -36,7 +36,7 @@
     <% if can? :transfer, document.id %>
       <li role="menuitem" tabindex="-1">
         <%= link_to(hyrax.new_work_transfer_path(document.id), class: 'itemicon itemtransfer', title: t("hyrax.dashboard.my.action.transfer")) do %>
-          <i aria-hidden="true" class='glyphicon glyphicon-transfer'></i>
+          <i aria-hidden="true" class="glyphicon glyphicon-transfer" aria-hidden="true"></i>
```

Comment:
aria-hidden is not needed twice here... if it is reported as an error, something else may be wrong. But should it possibly be added to glyphicon-star?